src/DynamoCoreWpf/Views/PackageManager/Controls/StepIndicatorControl.xaml.cs

```diff
@@ -0,0 +1,134 @@
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using System.Windows.Media;
+using Dynamo.UI;
+
+namespace Dynamo.PackageManager.UI
+{
+    /// <summary>
+    /// Interaction logic for StepIndicatorControl.xaml
+    /// </summary>
+    public partial class StepIndicatorControl : UserControl
+    {
+        private StepState _previousState;
+        private bool _clickedDuringMouseOver;
+
+        public StepIndicatorControl()
+        {
+            InitializeComponent();
+            this.UpdateVisualState(State);
+            this._clickedDuringMouseOver = false;
+        }
+
+        public static readonly DependencyProperty TitleProperty =
+         DependencyProperty.Register("Title", typeof(string), typeof(StepIndicatorControl), new PropertyMetadata("Step"));
+
+        public string Title
```

Comment:
Add summary to public properties

Reply:
Added summaries to all public properties!